server/container_create.go

```diff
@@ -56,6 +57,60 @@ func addOciBindMounts(sb *sandbox, containerConfig *pb.ContainerConfig, specgen
 	return nil
 }
 
+// buildOCIProcessArgs build an OCI compatible process arguments slice.
+func buildOCIProcessArgs(containerKubeConfig *pb.ContainerConfig, imageOCIConfig *v1.Image) []string {
+	processArgs := []string{}
+
+	kubeCommands := containerKubeConfig.Command
+	kubeArgs := containerKubeConfig.Args
+
+	if imageOCIConfig == nil {
+		// We did not get an OCI Image configuration.
+		// We should only use the information we got from kubelet,
+		// and fall back to a reasonable default otherwise.
```

Comment:
Should we? Docker isn't doing that and if no command is specified then the whole thing will blow up. Should we do the same?